guide.md

```diff
@@ -5,43 +5,62 @@ permalink: /guide/
 
 # Instructor guide
 
+## Why we teach this lesson
 
-### Core aspects
+Everyone should document their code, even if they’re working alone.
 
 These two are the most important points:
-- Documentation has to be versionnable and branchable
-- Documentation should be tracked together with the source code
+- Code documentation has to be versionnable and branchable
+- Code documentation should be tracked together with the source code
 
 **Please do not skim over the two above points**. Please take few minutes to
 explain why documentation (sources) should be tracked together with the source
 code.  Please discuss this aspect with workshop participants and connect it to
 **reproducibility**. This is for me (Radovan) the most important take-home
 message.
 
-Bring across that the tools we show are general:
-- Sphinx is not only for Python
-- Read the Docs is not only for Github
+Specific motivations:
 
+- Code documentation becomes quickly unmanageable if not part of the source code.
+- It helps people to quickly use your code thus reducing the time spent to explain over and again to new users.
+- It helps people to collaborate.
+- It improves the design of your code.
 
-### Place this lesson on last day of the workshop
+## Intended learning outcomes:
 
-Reason is that with collaborative Git we can create more interesting
-documentation exercises. Currently there are some elements of forking and
-pushing and this is only really introduced on day two.
+By the end of this lesson, learners should:
+- Understand the importance of writing code documentation together with the source code
+- Know what makes a good documentation
+- Learn what tools can be used for writing documentation
 
-We have tried this lesson on day one and it felt too early and disconnected/abrupt.
+## How to teach this lesson
 
-It works best after the reproducibility lesson since we then reuse the example
-and it feels familiar.
+### Requirements
 
+These are the software/packages used in the lesson material and exercises:
+- python 
+- Sphinx
 
-### What worked well in past workshops
+Attendees would need a Github account too.
 
-It is good if participants see how they can `sphinx-quickstart` a project from
-zero.
+### Typical pitfalls
 
-Then they learn how to deploy an existing project to RTD.
+#### Anaconda shell, Git, and Nano on Windows
 
+Windows users (probably) need to use the Anaconda prompt to run sphinx commands
+(sphinx-build, sphinx-quickstart). Git seems to be present in the standard
+anaconda installation, but not nano.
+
+A good working solution seems to be that Windows users have two terminals
+side-by-side, one with Anaconda prompt and another with git-bash.
+
+Another solution is to `conda install git` into an Anaconda prompt.
```

Comment:
is this supposed to be "git install nano"? because above it says "git seems to be present in the standard anaconda installation..."

Reply:
Yes . You are right. This is not very clear. More and more windows users install git from anaconda and VS Code editor (that is now installed along with Anaconda) for editing their files.

Having two windows open when using nano is probably the easiest and most effective way to work.